pkg/webreg/webreg.go

```diff
@@ -1228,6 +1229,138 @@ const imageExampleLiteral = `- as: repo-e2e
+  </tr>
+</table>
+
+<p>
+The process for granting a <i>lease</i> on a <i>resource</i> from the available pool generally follows this workflow:
+</p>
+
+<ul>
+  <li>a client (<i>lessee</i>) requests a <i>lease</i> on an available <i>resource</i></li>
+  <li>the server (<i>lessor</i>) grants the <i>lease</i>, if possible, or places the client in a FIFO queue to wait for the next available <i>resource</i></li>
+  <li>the client emits a heartbeat while the <i>lease</i> is under active use</li>
+  <li>the client relinquishes the <i>lease</i> once the it is no longer in use</li>
+  <li>the server places the <i>resource</i> into the available pool for future clients to request</li>
+</ul>
+
+<p>
+If a client fails to emit a heartbeat for long enough while the client holds a <i>lease</i>, the server will forcibly
+relinquish the <i>lease</i> and return the <i>resource</i> to the available pool for other clients. This mechanism
+ensures that clients which crash or otherwise fail to remain responsive cannot exhaust <i>resources</i> by holding a
+<i>lease</i> indefinitely.
+</p>
+
+<h3 id="admins"><a href="#admins">Directions for Cloud Administrators</a></h3>
+<p>
+An administrator of a cloud platform will interact with the leasing server in order to configure the aggregate limit on 
+jobs for the platform or inspect the current settings and usage. Care must be taken when configuring the leasing server
+in order to ensure that jobs are well-behaved against the cloud provider APIs.
+</p>
+
+<h4 id="adding"><a href="#adding">Adding A New Type Of Cloud Quota</a></h4>
+<p>
+In order to add a new type of cloud quota to the system, changes to the <code>boskos</code> leasing server configuration
+are required. The configuration is checked into source control <a href="https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml">here.</a>
+When adding a new type of quota, a new entry to the <code>resources</code> list is required, for example:
+</p>
+
+<code>boskos</code> configuration:
+{{ yamlSyntax (index . "dynamicBoskosConfig") }}
+
+<p>
+If it is not clear exactly how many concurrent jobs can share the cloud provider at once, the convention is to set the
+<code>min-count</code> and <code>max-count</code> to <code>1000</code>, to effectively leave jobs unlimited and allow
+for investigation.
+</p>
+
+<p>
+In addition to registering the volume of concurrent jobs that are allowed to against a new cloud platform, it is required
+that the leasing server is configured to reap leases which have not seen a recent heartbeat. This is done by adding the
+name of the resource type to the <a href="https://github.com/openshift/release/blob/master/core-services/prow/03_deployment/boskos_reaper.yaml#L27">reaper's configuration.</a>
+</p>
+
+<h5 id="static"><a href="#static">Configuration for Static, Limited Resources</a></h5>
```

Comment:
Dynamic resources are just as limited. We should make very clear here when this is required and when not, I don't think it has much to do if we are working against a `large cloud api`. IIUC its only about needing specific names for each individual resources to derive some data for it.

`Configuration when using boskos leases to derive data from for your build` maybe?